### PR TITLE
created class student

### PR DESCRIPTION
**Pull request by bekasatuboldiev**
добавлено функция создания студента

### CODE REVIEW

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/StudentController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/student")
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/StudentController.java:17]: api/students

File: src/main/java/kg/peaksoft/peaksoftlmsm1/db/entity/models/Course.java [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+import kg.peaksoft.peaksoftlmsm1.db.entity.User;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import java.util.Date;
+import java.util.List;
 
 @Data
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/db/entity/models/Course.java:12]: Use @Getter and #Setter instead @Data

**Review by ChyngyzIshen — COMMENTED** [review]
In StudentController class there is no @CrossOrogin and Swagger annotations

**Comment by bekasatuboldiev** [comment]
fixed

**Pull request merged**